app/icon_set.py

```diff
@@ -85,16 +85,6 @@ def get_icon(self, icon_name):
         """
         return Icon(f"{icon_name}", self)
 
-    def get_default_pixel_size(self):
```

Comment:
@ltshb  is there any reason why the icons of the default set should be 96 instead of 48 or can this be removed

Reply:
From what I remember, but @pakb might need to confirm, the initial idea was to deliver big icon and size them down on the viewer, which might be the cause of having 96px here. But now the new viewer is behaving like the old one which always retrieve the icons at scale 1x with size 48px and do the scaling in frontend.
We should make this service simple and for our current icon, we should rezize them on the repo to 48px, and only do a dynamic sizing if the scale is not 1x. The size in the metadata should always be the original size of the icon (at scale 1x), because looking closer at the spec, we cannot get icon metadata for a given scale.

Currently as it is it don't make much sense, icon are dynamicaly down sized on the backend and size up in the frontend !

Reply:
the main change that was previously done on `service-icons` is that we only keep the biggest size of each icon, and scale them down "on-the-fly" if needed. The caching would do the job of keeping the other sizes, instead of GitHub.
Prior to this change all sizes of each icons were pre-calculated and stored on GitHub. 

I agree with you @ltshb that we should stop changing size on the frontend and directly require the size we wanted in the first place.

Reply:
The way I understood @ltshb 's comment is, that he suggests to only store one size (for scale=1) of each icon in the repo and keep the dynamic rescaling in the frontend. 
Might makle sense, to keep the service simple. But in that case I'd suggest to keep the largest possible scale, not =1 but for scale = 2.5 or whatever is the largest scale we support.
Because with downscaling we would not have problems with decreasing quality, but for upscaling we might have those.

Reply:
Please keep stuff simple and don't try to optimize when it is not needed. Keep scaling on front end side by open layer which is more simple (ol is doing some computation with icons size and scale to match behavior of google earth, trying to do scaling ourself in backen will be much more complex and cause trouble in ol especially when mixing icons from our backend and external icons, believe me I tried it).
Again keep icon in backend to 48px scale 1. Up scaling them to 92px might be a bit of quality loss but probably not noticable and we anyway don't use it. Keep it stupid simple and don't optimize on assumption.

Reply:
As I said, let's have a call on this and no more comment ping pong!
There's potential for a lot of misunderstanding when doing that ping pong. I'll send an invitation, no more comments please